doc/source/contribute/user.rst

```diff
@@ -0,0 +1,119 @@
+Contributing as a user
+######################
+
+Users can contribute in a variety of ways, such as reporting bugs, requesting
+new features, testing in-development features, starting discussions, answering
+questions, and sharing their work with the community.
+
+.. warning::
+
+    Do not include any proprietary or sensitive information when reporting bugs
+    or showcasing your work.
+
+.. grid:: 1 1 3 3
+
+    .. grid-item-card:: :fa:`bug` Report bugs
+        :padding: 2 2 2 2
+        :link: report-bugs
+        :link-type: ref
+
+        Found a bug? Report it here.
+
+    .. grid-item-card:: :fa:`lightbulb` Request a new feature
+        :padding: 2 2 2 2
+        :link: request-a-new-feature
+        :link-type: ref
+
+        Got an idea for a new feature? Share it!
+
+    .. grid-item-card:: :fa:`vial-circle-check` Test a new feature
+        :padding: 2 2 2 2
+        :link: test-a-new-feature
+        :link-type: ref
+
+        Anxious to try out a new feature? Here's how you can do it.
+
+    .. grid-item-card:: :fa:`comments` Start a discussion
+        :padding: 2 2 2 2
+        :link: start-a-discussion
+        :link-type: ref
+
+        Want to discuss something? Start a discussion here.
```

Comment:
```suggestion
        Want to discuss something? Start or contribute to a discussion.
```
I assume that you'd want to sure a discussion on a topic doesn't already exist?

Reply:
Good catch!